src/coordination/azure/Akka.Coordination.Azure/AzureLease.cs

```diff
@@ -76,8 +75,9 @@ public AzureLease(ExtendedActorSystem system, AtomicBoolean leaseTaken, LeaseSet
 
             var client = new AzureApiImpl(system, azureLeaseSettings);
             
-            _leaseActor = system.ActorOf(
-                LeaseActor.Props(client, settings, _leaseName, leaseTaken),
+            _leaseActor = system.SystemActorOf(
```

Comment:
Moved the lease actor from user to system

Reply:
LGTM